### DISCUSSION TITLE
About GitHub Badges

### DISCUSSION THREAD

**Discussion by Luna-390** [community/community#20023] [Other Feature Feedback, Questions, & Ideas] [answered]
I am beginner of Git and GitHub. So I want to know how to achieve that  badges ,How do I get the yolo badge?

**Comment by Deshan555** [comment, marked as answer]
Why do people use GitHub badges?

Due of the quick approach that GitHub Badges provide users to collect repository metrics, they might improve the readability of GitHub repositories. In order to help other developers understand the value and need of your project, badges are intended to be placed in your README.md file.😎

**Reply by Luna-390** [reply]
So how about achieve that Badges ?

**Reply by Deshan555** [reply]
Pull Shark - Get Two PR's merged

Galaxy Brain - Answer at least 2 questions in discussions

YOLO - This one is really glitchy so really you might get a PR merged with a review and get the achievement, really weird.

for more info check that link 💯 : https://github.com/Schweinepriester/github-profile-achievements
